client/src/app/tabs/bpmn/modeler/BpmnModeler.js

```diff
@@ -39,19 +54,39 @@ export default class PlatformBpmnModeler extends BpmnModeler {
       }
     });
   }
+
+  getModules() {
```

Comment:
I changed this from overriding `_modules` so that the flag is used on runtime and not when the tests are bundled. This was required to create the test for the NewPropertiesProvider.